src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,13 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        UserService activeUserCheck = new UserService();
```

Comment:
```suggestion
        UserService userService = new UserService();
```
this is not some check, this is object of `UserService` class, so this is service (hence the name), let's call it accordingly